src/components/SidePanels/OutlinerPanel/OutlinerPanel.tsx

```diff
@@ -81,10 +86,19 @@ const OutlinerStandAlone: FC<OutlinerPanelProps> = ({ regions }) => {
         onFilterChange={onFilterChange}
       />
       {regions?.regions?.length > 0 ? (
-        <OutlinerTree
-          regions={regions}
-          selectedKeys={regions.selection.keys}
-        />
+        <>
+          <OutlinerTree
+            regions={regions}
+            selectedKeys={regions.selection.keys}
+          />
+          {hiddenRegions() > 0 && (
+            <Elem name="filters-empty">
+              <IconInfo width={21} height={20} />
+              <Elem name="filters-title">There {hiddenRegions() === 1 ? 'is' : 'are'} {hiddenRegions()} hidden region{hiddenRegions() > 1 && 's'}</Elem>
```

Comment:
```suggestion
              <Elem name="filters-title">There {hiddenRegions() === 1 ? 'is' : 'are'} {hiddenRegions()} hidden region{hiddenRegions() === 1 && 's'}</Elem>
```

Reply:
really the case where this would matter shouldn't happen since we shouldn't see the message if "0 hidden regions" would be displayed

Reply:
as mentioned this shouldnt happen as the message shouldnt appear when hidden region count is 0 so you can disregard this change